src/test/java/com/puppycrawl/tools/checkstyle/XpathFileGeneratorAuditListenerTest.java

```diff
@@ -269,6 +269,19 @@ public void testNoCloseStream() {
             .isEqualTo(0);
     }
 
+    @Test
+    public void testNoCloseStreamTwo() {
+        final XpathFileGeneratorAuditListener listener =
+                new XpathFileGeneratorAuditListener(outStream, null);
+        listener.finishLocalSetup();
+        listener.auditStarted(null);
+        listener.auditFinished(null);
+
+        assertWithMessage("Invalid close count")
+            .that(outStream.getCloseCount())
```

Comment:
If stream is not closed, try tif stream is not closed, try to write to stream some content , and validate presence of it